### PR TITLE
Taming the savage contraction issue

### LINKED ISSUE #5215: Our "little" contraction problem
Refactor topics to use contractions.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5215 🐲 

I scanned these, and only a few reversions were required (e.g., "it'sn't" :point_right: back to "it isn't"). I'll run my eye over these again tomorrow (Friday).

In the process, I spotted a couple of minor spelling nits in ANCM error messages that you may see too on review. I sent them a PR: https://github.com/aspnet/AspNetCoreModule/pull/286

This PR addresses the `aspnetcore` doc set. If you'd like to see one of these for the `aspnet` doc set, let me know.

### CODE REVIEW

**Comment by guardrex** [comment]
Hold review for sec ... I still need to review this and make one more small update.

**Comment by guardrex** [comment]
I made a few passes to get the reversions out (e.g., error messages don't use contractions). It should be darn close to correct. Sorry for the eye strain on review. We'll be very close to a consistent contractions presentation after we get through this.

File: aspnetcore/data/ef-rp/complex-data-model.md [hunk 1]
```diff
@@ -639,7 +639,7 @@ With the preceding changes, existing `Course` rows will be related to the "Temp"
 A production app would:
 
 * Include code or scripts to add `Department` rows and related `Course` rows to the new `Department` rows.
-* Would not use the "Temp" department or the default value for `Course.DepartmentID `.
+* Wouldn't use the "Temp" department or the default value for `Course.DepartmentID `.
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/complex-data-model.md]: Wouldn't --> Not

File: aspnetcore/data/ef-rp/intro.md [hunk 2]
```diff
@@ -323,19 +323,19 @@ In the following code, the `async` keyword, `Task<T>` return value, `await` keyw
...
 
 * `ToListAsync` is the asynchronous version of the `ToList` extension method.
 
 Some things to be aware of when writing asynchronous code that uses EF Core:
 
-* Only statements that cause queries or commands to be sent to the DB are executed asynchronously. That includes, `ToListAsync`, `SingleOrDefaultAsync`, `FirstOrDefaultAsync`, and `SaveChangesAsync`. It does not include statements that just change an `IQueryable`, such as `var students = context.Students.Where(s => s.LastName == "Davolio")`.
+* Only statements that cause queries or commands to be sent to the DB are executed asynchronously. That includes, `ToListAsync`, `SingleOrDefaultAsync`, `FirstOrDefaultAsync`, and `SaveChangesAsync`. It doesn't include statements that just change an `IQueryable`, such as `var students = context.Students.Where(s => s.LastName == "Davolio")`.
 
-* An EF Core context is not threaded safe: don't try to do multiple operations in parallel. 
+* An EF Core context isn't threaded safe: don't try to do multiple operations in parallel.
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/intro.md]: Unrelated to your changes, but let's fix it while you're in here. Change "threaded" to "thread".

**Comment by guardrex** [comment]
ty @scottaddie ... The eye drops are on ME! :smile:

**Comment by Rick-Anderson** [comment]
@guardrex your /en-us purge comes back to bite you.

**Comment by Rick-Anderson** [comment]
@guardrex  had to fix it twice.

**Comment by guardrex** [comment]
A few days ago, I sent GH a message regarding it not being fixed. They re-opened their issue on it.

**Comment by scottaddie** [comment]
@guardrex There's also a merge conflict to be resolved. 👇

**Comment by scottaddie** [comment]
@guardrex Thanks for taking care of this! My eyes are still burning. 😄

**Pull request merged**